components/avatar.css

```diff
@@ -0,0 +1,57 @@
+
+.avatar{
+    border: 2px solid var(--secondary-colour);    
+    border-radius: 50%;
+}
+
+.avatar-sm{
+    width: 30px;
+    height: 30px;
+}
+
+.avatar-md{
+    width: 35px;
+    height: 35px;
+}
+
+.avatar-lg{
+    width: 45px;
+    height: 45px;
+}
+
+.avatar-xl{
+    width: 110px;
+    height: 110px;
+}
+
+.avatar-text{
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    text-transform: uppercase;
+    background-color: var(--quaternary-colour);
+}
+
+.text-sm{
+    font-size: 13px;
+    width: 30px;
+    height: 30px;
```

Comment:
Try to use relative unit instead of absolute unit

Reply:
Have changed it today, will update it by tomorrow.